native/iparmq.go

```diff
@@ -69,8 +69,11 @@ func (Implementation) Iparmq(ispec int, name, opts string, n, ilo, ihi, lwork in
 
 	case 14:
 		// Skip a multi-shift QR iteration whenever aggressive early
-		// deflation finds at least (nibble*(window size)/100)
-		// deflations.
+		// deflation finds at least nibble*(window size)/100 deflations.
+		// The default, small value reflects the expectation that the
```

Comment:
I added a closing comma and reorganized the whole comment to make the second sentence shorter.